src/org/yccheok/jstock/gui/JStock.java

```diff
@@ -4425,11 +4445,32 @@ public void mouseReleased(MouseEvent e) {
 
         private void maybeShowPopup(MouseEvent e) {
             if (e.isPopupTrigger()) {
+                if (jTable1.getSelectedRowCount() <= 1) {
+                   SetFocusToRightClickLocation(e, jTable1); 
+                }
                 if (jTable1.getSelectedRowCount() > 0) {
                     getMyJTablePopupMenu().show(e.getComponent(), e.getX(), e.getY());
                 }
             }
         }
+
+        private void SetFocusToRightClickLocation(MouseEvent e, JTable table) {
```

Comment:
I was talking on `SetFocusToRightClickLocation`

But I had fixed it : https://github.com/yccheok/jstock/blob/master/src/org/yccheok/jstock/gui/JStock.java#L4448